src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import 'blocks/variables';
+@import 'blocks/card';
+@import 'blocks/stars';
+
+* {
+  box-sizing: border-box;
   margin: 0;
```

Comment:
only  box-sizing: border-box; is allowed to use for `*` selector. Reset margin for those tags you need below